src/prefect/telemetry/run_telemetry.py

```diff
@@ -50,15 +51,19 @@ class RunTelemetry:
         default_factory=lambda: get_tracer("prefect", prefect.__version__)
     )
     span: Span | None = None
+    _enabled: bool = field(
+        default_factory=lambda: prefect.settings.get_current_settings().cloud.enable_orchestration_telemetry
+    )
 
     async def async_start_span(
         self,
         run: FlowOrTaskRun,
         client: PrefectClient,
         parameters: dict[str, Any] | None = None,
-    ) -> Span:
+    ) -> Optional[Span]:
```

Comment:
```suggestion
    ) -> Span | None:
```

Reply:
worth mentioning there'd be an unused import if these suggestions were just accepted